.github/workflows/cont_integration.yml

```diff
@@ -41,22 +41,16 @@ jobs:
       - name: Pin dependencies for MSRV
         if: matrix.rust.version == '1.63.0'
         run: |
-          cargo update -p zstd-sys --precise "2.0.8+zstd.1.5.5"
-          cargo update -p time --precise "0.3.20"
-          cargo update -p home --precise "0.5.5"
-          cargo update -p proptest --precise "1.2.0"
-          cargo update -p url --precise "2.5.0"
-          cargo update -p cc --precise "1.0.105"
           cargo update -p tokio --precise "1.38.1"
           cargo update -p tokio-util --precise "0.7.11"
-          cargo update -p indexmap --precise "2.5.0"
-          cargo update -p security-framework-sys --precise "2.11.1"
-          cargo update -p csv --precise "1.3.0"
-          cargo update -p unicode-width --precise "0.1.13"
           cargo update -p rustls@0.23.20 --precise "0.23.19"
+          cargo update -p url --precise "2.5.0"
+          cargo update -p security-framework-sys --precise "2.11.1"
+          cargo update -p indexmap --precise "2.5.0"
       - name: Build
-        run: cargo build --workspace --exclude 'example_*' ${{ matrix.features }}
+        run: cargo build --workspace --exclude 'example_*' --exclude 'bdk_testenv' ${{ matrix.features }}
```

Comment:
Since we are keeping such an old MSRV and our test deps change so often I think this is the only way to manage it. But I'm open to move this to a different PR for after 1.0 if this is going to need more discussion.